### ISSUE TITLE
Calendar popup not responsive for smaller resolutions

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1280]
### Description

When I want to add a new event on my 14" laptop, the popup is not responsive and I can't choose later dates in the month or change the hours if I don't zoom out on the page.
<img width="960"  src="https://github.com/cryptpad/cryptpad/assets/95225431/0496715a-4114-4043-b97e-1bd6d91ca869">

### Steps to reproduce

1. get a device that has 14" or less
2. go to calendar and try to add a new event

### Expected behavior

Ideally the popups should be responsive for every screen size

### What opreating system are you using?

Microsoft Windows

### What browsers are you seeing the problem on?

Mozilla Firefox

### Extensions

.

### Version

5.5.0

**Comment by ansuz** [comment]
I noticed that this is also the case for the dropdown for repeating events:
![Screenshot from 2023-11-05 11-09-59](https://github.com/cryptpad/cryptpad/assets/1264398/8ada5fff-567a-4f09-b943-56d9e557876d)

removing `overflow-y: auto;` from [app-calendar.less](https://github.com/cryptpad/cryptpad/blob/main/www/calendar/app-calendar.less#L130) seems to address this issue:

![Screenshot from 2023-11-05 11-10-11](https://github.com/cryptpad/cryptpad/assets/1264398/5f3a648c-7893-40a3-829c-cd797d05f475)

```diff
diff --git a/www/calendar/app-calendar.less b/www/calendar/app-calendar.less
index cab5ab0c2..b0b9f7da8 100644
--- a/www/calendar/app-calendar.less
+++ b/www/calendar/app-calendar.less
@@ -126,9 +126,9 @@
                 .tui-full-calendar-popup {
                     width: 540px;
                     max-width: 100%;
                     max-height: 100%;
-                    overflow-y: auto;
+                    //overflow-y: auto;
                 }
             }
             #tui-full-calendar-popup-arrow {
                 display: none !important;
```

...but I haven't tested thoroughly to confirm that it won't cause other problems.
